src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/CodeActions/RazorCodeActionRunner.ts

```diff
@@ -0,0 +1,38 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import * as vscode from 'vscode';
+import { RazorLanguageServerClient } from '../RazorLanguageServerClient';
+import { RazorLogger } from '../RazorLogger';
+import { CodeActionResolutionRequest } from '../RPC/CodeActionResolutionRequest';
+import { CodeActionResolutionResponse } from '../RPC/CodeActionResolutionResponse';
+import { convertWorkspaceEditFromSerializable } from '../RPC/SerializableWorkspaceEdit';
+
+export class RazorCodeActionRunner {
+
+    constructor(
+        private readonly serverClient: RazorLanguageServerClient,
+        private readonly logger: RazorLogger,
+    ) {}
+
+    public register() {
+        vscode.commands.registerCommand('razor/runCodeAction', (request: CodeActionResolutionRequest) => this.runCodeAction(request), this);
```

Comment:
Need to return this command so the disposable can be disposed when the extension shuts down